tutorials/text/text_intro.py

```diff
@@ -31,36 +32,38 @@
 ===================
 
 The following commands are used to create text in the pyplot
-interface
+interface and the object-oriented API:
```

Comment:
I think this would be better presented as a 3-column table (pyplot, oo, explanation).

Reply:
I agree, but making a table that will fit is not trivial for someone w/ my rst skills.  I think the list is OK.